content/400-reference/200-api-reference/200-command-reference.mdx

```diff
@@ -571,229 +569,44 @@ Push the schema with a custom schema location:
-  provider = "sqlite"
-  url      = "file:my-database.db"
-}
-```
+### <inlinecode>migrate resolve</inlinecode>
 
 #### Arguments
 
-The point to migrate back to can be defined in any of the following three ways:
-
-| Argument  | Required | Description                                                                             | Default |
-| --------- | -------- | --------------------------------------------------------------------------------------- | ------- |
-| decrement | No       | Specifies the number of backwards migrations to apply.                                  | 1       |
-| name      | No       | Specifies where to migrate back to using the name of the final migration to apply.      |
-| timestamp | No       | Specifies where to migrate back to using the timestamp of the final migration to apply. |
-
-#### Options
-
-Additionally, the following options modify the behavior of the `migrate down` command:
-
-| Option            | Required | Description                                                                                                                                         | Default                                     |
-| ----------------- | -------- | --------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------- |
-| `--experimental`  | Yes      | Enables use of Experimental commands                                                                                                                |                                             |
-| `-p`, `--preview` | No       | Preview the migration that would be created without writing any changes to the filesystem.                                                          |                                             |
-| `--schema`        | No       | Specifies the path to the desired `schema.prisma` file to be processed instead of the default path. Both absolute and relative paths are supported. | `./schema.prisma`, `./prisma/schema.prisma` |
+| Argument   | Required | Description                                                                                                                                       | Default                                          |
+| :--------- | :------- | :------------------------------------------------------------------------------------------------------------------------------------------------ | :----------------------------------------------- |
+| `--applied` | No       | Specifies the path to the desired schema.prisma file to be processed instead of the default path. Both absolute and relative paths are supported. | `./schema.prisma`<br /> `./prisma/schema.prisma` |
+| `--rolledback` | No       | Specifies the path to the desired schema.prisma file to be processed instead of the default path. Both absolute and relative paths are supported. | `./schema.prisma`<br /> `./prisma/schema.prisma` |
```

Comment:
Flag was renamed from `--rolledback` to `--rolled-back`
```suggestion
| `--rolled-back` | No       | Specifies the path to the desired schema.prisma file to be processed instead of the default path. Both absolute and relative paths are supported. | `./schema.prisma`<br /> `./prisma/schema.prisma` |
```